config/pmd/ruleset.xml

```diff
@@ -12,6 +12,8 @@
   </rule>
   <rule ref="category/java/codestyle.xml">
     <exclude name="AtLeastOneConstructor" />
+    <exclude name="CommentDefaultAccessModifier" />
+    <exclude name="DefaultPackage" />
```

Comment:
why?

Reply:
Since usage of newer libraries, there is no need to provide `public` accessors for certain methods like `Test` cases or `setUp` methods. PMD is not aware of that and complains in vain

Reply:
So only in tests then?
Perhaps this can be addressed across the project separately from this use story, after it's discussed?